client/my-sites/email/titan-add-mailboxes/titan-mailbox-pricing-notice.jsx

```diff
@@ -23,6 +23,62 @@ const doesAdditionalPriceMatchStandardPrice = ( domain, titanMonthlyProduct ) =>
 	);
 };
 
+function getPriceMessage( props ) {
+	const { purchaseCost, translate } = props;
+
+	return purchaseCost.amount === 0
+		? translate( 'You can add new mailboxes for free until the end of your trial period.' )
+		: translate(
+				'You can purchase new mailboxes at the prorated price of {{strong}}%(proratedPrice)s{{/strong}} per mailbox.',
+				{
+					args: {
+						proratedPrice: purchaseCost.text,
+					},
+					components: {
+						strong: <strong />,
+					},
+					comment:
+						'%(proratedPrice)s is a formatted price for an email subscription (e.g. $3.50, €3.75, or PLN 4.50)',
+				}
+		  );
+}
+
+function getPriceMessageExplanation( props ) {
+	const { purchaseCost, renewalCost, translate } = props;
+
+	//We don't need any explanation of the price at this point, because we have already handled it previusly.
```

Comment:
Typo: previusly => previously
```suggestion
	// We don't need any explanation of the price at this point, because we have already handled it previously.
```